coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -13,7 +13,30 @@
         <v-expansion-panel-header>
           <div class="list-item">
             <div class="filing-type">
-              <div class="list-item__title mb-1">{{item.name}}</div>
+              <v-row>
+                <v-col cols="6" class="v-col-padding">
+                  <div class="list-item__title mb-1">{{item.name}}</div>
+                </v-col>
+                <v-col cols="6" class="v-col-padding" v-if="isCoaFutureEffective(item.name, item.status)">
+                  <v-scale-transition>
+                    <v-tooltip
+                      top
+                      content-class="pending-tooltip"
+                    >
+                      <template v-slot:activator="{ on }">
+                        <div id="pending-alert" class="list-item__subtitle" v-on="on">
+                          <v-icon color="#ffeb3b" small>mdi-alert</v-icon>
+                          PAID AND PENDING
+                        </div>
+                      </template>
+                      <span>
+                      The updated office addresses will be legally effective on {{ item.filingEffectiveDate }},
+                      12:01 AM(Pacific Time). No other filings are allowed until then.
```

Comment:
Need a space between "AM" and "(Pacific".

Reply:
Thank you, will do!